cypress/page/verify-step.page.ts

```diff
@@ -0,0 +1,12 @@
+class VerifyPage {
+    private confirmationText: string;
+
+    constructor() {
+      this.confirmationText = "#center_column > div > p > strong";
+    }
+
+    public verifyPurchase(): void {
+      cy.get(this.confirmationText).should("have.text", "Your order on My Store is complete.");
```

Comment:
A best practice is to not perform assertions on page objects, so, it will be better to make this method return the confirmation title, change the name of it as getConfirmationTitle or something like that, and perform the assertion on the test, that is the cypress/integration/buy-shirt.spec.ts file